e2e/test/scenarios/question/questions-entity-id.cy.spec.ts

```diff
@@ -41,6 +41,9 @@ describe("scenarios > questions > entity id support", () => {
     // await the entity id request to make sure the "wrong" request had its time to get fired
     cy.wait("@entityIdRequest");
 
+    // it should render a 404 page as that entity id doesn't exist
+    H.main().findByText("We're a little lost...").should("be.visible");
```

Comment:
This test is from https://github.com/metabase/metabase/pull/52892, I updated it to check that it's rendering a 404 page as it should